backend/sirius-web-spring-collaborative-diagrams/src/main/resources/schema/diagram.graphqls

```diff
@@ -270,7 +270,7 @@ input InvokeNodeToolOnDiagramInput {
   editingContextId: ID!
   representationId: ID!
   selectedObjectId: String
-  diagramElementId: ID!
+  diagramElementId: ID
```

Comment:
Do not change that, that is not related to the issue at hand

Reply:
Done